bundles/best_practices.rst

```diff
@@ -81,7 +81,9 @@ The following is the recommended directory structure of an AcmeBlogBundle:
 This directory structure requires to configure the bundle path to its root
 directory as follows::
 
-    class AcmeBlogBundle extends Bundle
+    use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
+
+    class AcmeBlogBundle extends AbstractBundle
     {
         public function getPath(): string
```

Comment:
AFAIK, when we extend AbstractBundle, no need to override this. Maybe we can remove it ?

Reply:
Good catch! I committed some changes to explain this differently. Thanks!